### DISCUSSION TITLE
new error with phpfastcache that we are unable to track down.

### DISCUSSION THREAD

**Discussion by radamnyc** [TruCopilot/phpfastcache#791] [Q&A]
Hello,   as of noon yesterday we started seeing an error with phpfastcache that has had us stumped for 10 hours.  so much so we had to remove phpfastcache from production.  we are using version 5.0.21.  here are the logs and the error we are getting, and help would be greatly appreciated.  

[Sat Mar 20 20:45:39.684117 2021]  Loading  phpFastCache\\CacheManager
[Sat Mar 20 20:45:39.684188 2021]  Loading  Symfony\\Component\\HttpKernel\\KernelInterface
[Sat Mar 20 20:45:39.684245 2021]  Loading  phpFastCache\\Drivers\\Memcached\\Driver
[Sat Mar 20 20:45:39.684288 2021]  Loading  phpFastCache\\Core\\DriverAbstract
[Sat Mar 20 20:45:39.684326 2021]  Loading  phpFastCache\\Cache\\DriverBaseTrait
[Sat Mar 20 20:45:39.684370 2021]  Loading  phpFastCache\\Core\\ExtendedCacheItemPoolTrait
[Sat Mar 20 20:45:39.684414 2021]  Loading  phpFastCache\\Core\\StandardPsr6StructureTrait
[Sat Mar 20 20:45:39.684682 2021]  Loading  phpFastCache\\Core\\ClassNamespaceResolverTrait
[Sat Mar 20 20:45:39.684825 2021]  Loading  phpFastCache\\Cache\\ExtendedCacheItemPoolInterface
[Sat Mar 20 20:45:39.684905 2021]  Loading  Psr\\Cache\\CacheItemPoolInterface
[Sat Mar 20 20:45:39.684982 2021]  Loading  phpFastCache\\Core\\MemcacheDriverCollisionDetectorTrait
[Sat Mar 20 20:45:39.685061 2021]  Loading  Memcached
[Sat Mar 20 20:45:39.685193 2021]  Loading  phpFastCache\\Exceptions\\phpFastCacheDriverCheckException
[Sat Mar 20 20:45:39.685271 2021]  Loading  phpFastCache\\Exceptions\\phpFastCacheDriverException
[Sat Mar 20 20:45:39.685728 2021]  PHP Fatal error:  Uncaught phpFastCache\\Exceptions\\phpFastCacheDriverCheckException: Memcached is not installed or is misconfigured, cannot continue. in /var/www/smaw/vendor/phpfastcache/phpfastcache/src/phpFastCache/Drivers/Memcached/Driver.php:45\nStack trace:\n#0 /var/www/smaw/vendor/phpfastcache/phpfastcache/src/phpFastCache/CacheManager.php(107): phpFastCache\\Drivers\\Memcached\\Driver->__construct(Array)\n#1 /var/www/smaw/lib/utilities/splickitcache.php(147): phpFastCache\\CacheManager::getInstance('Memcached', Array)

**Comment by Geolim4** [comment]
Hello,


Just read the error message and try to fix out the issue 🤷🏻‍♂️ 

Also this version is not longer supported because it reached EOL.

Cheers,
Georges
